buildSrc/src/main/java/org/elasticsearch/gradle/tool/DockerUtils.java

```diff
@@ -165,11 +155,19 @@ public static void assertDockerIsAvailable(Project project, List<String> tasks)
             throwDockerRequiredException(message);
         }
 
+        final String operation;
+        if (availability.version == null) {
```

Comment:
This seems unnecessary to me. This final message is meant to be a catch-all anyway so I don't think there's much benefit in trying to be any more specific in the error message.

Reply:
It was helpful to me to figure out which command docker is failing on my machine, so I can run this command and see if it works or not. I am ok with removing it, since the error message was more helpful anyway.

Reply:
If we consider this valuable I'd prefer we wire the actual command through `Result` and just display _that_ in this error rather than a simple hard-coded description.